metrics/ci_integrations/lib/cli/parties/configured_parties/configured_destination_party.dart

```diff
@@ -0,0 +1,20 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:ci_integration/cli/parties/configured_parties/configured_party.dart';
```

Comment:
It seems like the `configure_parties` should lay on the same level as a `parties` package considering the [design document](https://github.com/platform-platform/monorepo/blob/master/metrics/ci_integrations/docs/05_ci_integration_commands_code_sharing.md#general-class-diagram).